scala/src/main/scala/smile/validation/MutualIndex.scala

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
```

Comment:
Three class design is over-complicated. A single class is enough. The default `normalization` parameter in constructor can be `None`.